docs/index.rst

```diff
@@ -12,7 +12,7 @@ performing intelligent retries. Install it via
 
 .. note::
 
-   The Databricks SDK for Python is in an Experimental state. To provide feedback, ask questions, and report issues, use the Issues tab in the Databricks SDK for Python repository in GitHub.
+   The Databricks SDK for Python is in an Experimental state. To provide feedback, ask questions, and report issues, use the `Issues tab <https://github.com/databricks/databricks-sdk-py/issues>`_ in the `Databricks SDK for Python repository <https://github.com/databricks/databricks-sdk-py>`_ on GitHub.
```

Comment:
can you move it outside of the note - we'll be receiving feedback through issues ever after this disclaimer removed :)